apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -17,7 +17,13 @@
 using Google.Protobuf.WellKnownTypes;
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Diagnostics;
+#if !NETSTANDARD1_5
+// TODO: Remove when we support .NET 4.6.1
```

Comment:
I also cleaned up some of the conditional compilation blocks, but let me know if you want me to reorganize anything.